### PR TITLE
Tell us more about the new job

### PR DESCRIPTION
**Pull request by ondrae**
screen shot 2018-11-12 at 4 59 43 pm
screen shot 2018-11-12 at 5 01 08 pm

### CODE REVIEW

File: spec/forms/tell_us_about_the_new_job_form_spec.rb [hunk 1]
```diff
@@ -35,22 +31,20 @@
...
-        it "is invalid" do
-          invalid_params = valid_params.merge(manager_phone_number: nil)
-          form = TellUsAboutTheNewJobForm.new(nil, invalid_params)
+        it "is valid" do
+          form = TellUsAboutTheNewJobForm.new(nil, valid_params.merge(manager_phone_number: nil))
 
-          expect(form).to_not be_valid
+          expect(form).to be_valid
         end
       end
```
Comment (hartsick) [inline spec/forms/tell_us_about_the_new_job_form_spec.rb]: can we add a test in here for submitting an empty string as the manager phone number, since that's the behavior we're seeing? 

here's an example I did from the similar branch:
```
      context "when the manager_phone_number is a blank string" do
        it "is valid" do
          params = valid_params.merge(manager_phone_number: "")
          form = TellUsAboutTheLostJobForm.new(nil, params)
          expect(form).to be_valid
        end
      end
```

File: app/views/tell_us_about_the_new_job/edit.html.erb [hunk 2]
```diff
@@ -3,19 +3,19 @@
 <% content_for :card_body do %>
   <%= fields_for(:form, @form, builder: CfaFormBuilder) do |f| %>
     <%= f.cfa_input_field :company_name, "What is the name of the company?" %>
+
+    <div class="notice">
+      <p>The following questions are optional, but they'll help the county verify your change. Try to answer them.</p>
```
[the remark this answers is not in the document]
Comment (hartsick) [reply app/views/tell_us_about_the_new_job/edit.html.erb]: Rachel says we can keep it the same

File: app/forms/tell_us_more_about_the_new_job_form.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class TellUsMoreAboutTheNewJobForm < Form
+  set_attributes_for :change_report, :first_day_year, :first_day_month, :first_day_day, :paid_yet
+
+  set_attributes_for :navigator, :street_address, :city, :zip_code
```
[the remark this answers is not in the document]
Comment (ondrae) [reply app/forms/tell_us_more_about_the_new_job_form.rb]: gone

File: spec/features/report_new_job_spec.rb [hunk 4]
```diff
@@ -53,6 +53,9 @@
       fill_in "What is the name of the company?", with: "Abc Corp"
       fill_in "What is the name of Jane's supervisor or manager?", with: "My boss"
       fill_in "What is this person's phone number?", with: "999-888-7777"
+      proceed_with "Continue"
+
+      expect(page.title).to eq "Tell us more about the new job. | ReportChangesColorado.org"
```
[the remark this answers is not in the document]
Comment (ondrae) [reply spec/features/report_new_job_spec.rb]: done

File: spec/forms/tell_us_about_the_new_job_form_spec.rb (hunk 1 above)
Comment (ondrae) [reply spec/forms/tell_us_about_the_new_job_form_spec.rb]: Good point, used your example.

File: app/views/tell_us_about_the_new_job/edit.html.erb (hunk 2 above)
[the remark this answers is not in the document]
Comment (ondrae) [reply app/views/tell_us_about_the_new_job/edit.html.erb]: done

File: spec/forms/tell_us_about_the_new_job_form_spec.rb [hunk 5]
```diff
@@ -146,32 +79,6 @@
...
       expect(change_report.manager_name).to eq "Boss McBosser"
       expect(change_report.manager_phone_number).to eq "1112223333"
-      expect(change_report.first_day.year).to eq 2000
-      expect(change_report.first_day.month).to eq 1
-      expect(change_report.first_day.day).to eq 15
-      expect(change_report.paid_yet_yes?).to be_truthy
-    end
-  end
-
-  describe ".from_change_report" do
```
[the remark this answers is not in the document]
Comment (ondrae) [reply spec/forms/tell_us_about_the_new_job_form_spec.rb]: kept it
